pyQuARC/main.py

```diff
@@ -6,19 +6,20 @@
 
 from tqdm import tqdm
 
-from pyQuARC.code.utils import get_headers
-
 if __name__ == '__main__':
+    from code.utils import get_headers
     from code.checker import Checker
     from code.constants import COLOR, ECHO10_C, SUPPORTED_FORMATS
     from code.downloader import Downloader
     from code.utils import get_cmr_url, is_valid_cmr_url
```

Comment:
add get_headers to the list of methods listed here?

Reply:
This appears to be a mistake. Lines 10 and 17 should be removed.

Reply:
This hasn't been fixed yet?